src/react/workflow/ExpirationForm.tsx

```diff
@@ -198,6 +224,33 @@ export function ExpirationForm({ prefix = '' }: Props) {
               />
             }
           />
+
+          {isISVBucket && (
+            <Stack direction="vertical" gap="r16">
+              <Banner
+                icon={<Icon name="Exclamation-circle" color="statusWarning" />}
+                variant="warning"
+              >
+                This bucket is tagged as being used in a ISV use-case.
```

Comment:
@Cuervino I think ISV is not the public word we use, better use Connector here no, wdyt ?
```suggestion
                This bucket is tagged as being used in a connector use-case.
```